apis/python/src/tiledb/vector_search/ingestion.py

```diff
@@ -2948,6 +2956,8 @@ def consolidate_and_vacuum(
             group.meta["partition_history"] = json.dumps(partition_history)
             group.meta["base_sizes"] = json.dumps(base_sizes)
             group.meta["ingestion_timestamps"] = json.dumps(ingestion_timestamps)
+            group.meta["distance_metric"] = int(distance_metric)
```

Comment:
The code here is not called for all index algorithms. Move this inside the `create` function of each algorithm

Reply:
Sure we could add it in `create_metadata`. However, we would also need to check wether each algorithm supports or not the specified distance